scenarios/search-menu.json

```diff
@@ -43,6 +43,10 @@
       "url": "{url}/civicrm/contact/search/builder?reset=1",
       "hideSelectors": ["#mainTabContainer"]
     },
+    {
+      "label": "Search Builder - Search result",
+      "url": "{url}/civicrm/contact/search/builder?_qf_Builder_display=true&qfKey=fe63cdd9a7bc6d5d41dd1436198fac34_3645"
```

Comment:
i don't trust this url, it contains hardcoded values. Please write an onReady script